### ISSUE TITLE
Gutenberg: add classic editor escape hatch for Calypsoify

### ISSUE THREAD

**Issue by gwwar** [Automattic/wp-calypso#28207]
Let's make sure folks can update their editor preference preference for Simple Sites!

For now this should live in the options dropdown:

![screen-shot-2018-10-29-at-11-57-53-am](https://user-images.githubusercontent.com/1270189/47823661-884c5600-dd26-11e8-855e-ae79d9d9db9a.png)

**Comment by gwwar** [comment]
@mmtr, @alexislloyd suggested that we move this to the Editor section above, which may make more sense. Maybe we can also add an external link icon to note that this will trigger a navigation event?

**Comment by gwwar** [comment]
Good point on the inconsistency. I'll defer to our designer friends @shaunandrews @alexislloyd 

Don't let this block you for now @mmtr!

**Comment by kwight** [comment]
In progress in D20254-code.

**Comment by shaunandrews** [comment]
<img width="1260" alt="screen shot 2018-11-06 at 12 38 23 pm" src="https://user-images.githubusercontent.com/191598/48082533-e5318b80-e1c0-11e8-8e79-cc0d2e0ad36f.png">

I tried a few ideas. I started by going with a button, since this does cause you to navigate away from the editor. (Sidenote: Should we have a confirmation/warning here?) I think the button works at the bottom, but its a little too prominent.

I then tried adding the option to the Editor grouping with an icon. This feels super weird, even if we find a suitable icon.

The last one (and my favorite) is to just add the simple text to the bottom, below options.
